interfaces/cython/cantera/ck2yaml.py

```diff
@@ -865,6 +865,14 @@ def read_NASA7_entry(self, lines, TintDefault, comments):
         if not composition:
             raise InputError("Error parsing elemental composition for "
                              "species '{}'", species)
+        else:
+            for symbol in composition.keys():
+                # Here it checks if there is a number in the symbol string and if this is
+                # in the element list as a customized element
+                if any(map(str.isdigit, symbol)) == True and symbol not in self.elements:
+                    raise InputError("Error parsing elemental composition for "
+                             "species thermo entry\n{}"
+                             "\n the first line has an incorrect format", "".join(lines))
```

Comment:
Can you modify the formatting of these lines so that they conform to PEP 8? Thanks 😄